server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/SessionRoutes.scala

```diff
@@ -73,6 +75,7 @@ class SessionRoutes @Inject()(@Named(InjectionKeys.RFC_8621) val authenticator:
       .flatMap(session => sendRespond(session, response))
       .onErrorResume(throwable => SMono.fromPublisher(errorHandling(throwable, response)))
       .asJava()
+      .doOnSuccess(_ => httpClientMetrics.update())
```

Comment:
Trigger the polling update upon successful GET `/jmap/session` seems ok IMO. While it could be a bit much doing it at POST `/jmap` (or we could add some random).